.eslintrc.json

```diff
@@ -1,3 +1,9 @@
 {
-  "extends": "./node_modules/gts"
+  "extends": "./node_modules/gts",
+  "rules": {
```

Comment:
I believe this file is still autogenerated in `synth.py` nightly, you can just add an `excludes` on `templates.copy` sorry for the nuisance.